test/models/badges/functional_february_badge_test.rb

```diff
@@ -0,0 +1,52 @@
+require "test_helper"
+
+class Badges::FunctionalFebruaryBadgeTest < ActiveSupport::TestCase
+  test "attributes" do
+    badge = create :functional_february_badge
+    assert_equal "Functional February", badge.name
+    assert_equal :ultimate, badge.rarity
```

Comment:
It this `ultimate` or `rare`? It's hard to determine this beforehand I feel.

Reply:
Yeah, we can maybe change it later?